tkosubs.go

```diff
@@ -24,14 +24,16 @@ func main() {
 	gotenv.Load()
 
 	domainsFilePath := os.Args[1]
+	recordsFilePath := os.Args[2]
+	outputFilePath := os.Args[3]
```

Comment:
@mhmdiaa it would be nice to get these by specifying flags. would make it easier to integrate in other tools/frameworks